config/v1/types_ingress.go

```diff
@@ -54,6 +54,22 @@ type IngressSpec struct {
 	// configurable routes.
 	// +optional
 	ComponentRoutes []ComponentRouteSpec `json:"componentRoutes,omitempty"`
+
+	// requiredHSTSPolicies specifies HSTS policies that are required to be set on
+	// newly created routes matching criteria that are specified in the policy.
+	// - If there are no RequiredHSTSPolicies, any route annotation is valid.
```

Comment:
this is specifically the hsts annotation, right?

Reply:
Right.  I'll clarify.